File: netflix2/java/src/main/java/repository/BookingDao.java
package repository;

import source.DB;

public class BookingDao {

    public static void book(String dvdId, int quantity, String userId) {
        DB.insert(String.format("INSERT INTO BOOKINGS(DVD_ID, QUANTITY, USER_ID) values('%s', %s, '%s')", dvdId, quantity, userId));
    }
}

File: netflix2/csharp/Netflix2/repository/BookingDao.cs
namespace Netflix2.repository
{
    public static class BookingDao
    {
        public static bool Book(string dvdId, int quantity, string userId)
        {
            return DB.Insert($"INSERT INTO BOOKINGS(DVD_ID, QUANTITY, USER_ID) values('{dvdId}', {quantity}, '{userId}')");
        }
    }
}